conans/tools.py

```diff
@@ -684,6 +693,7 @@ def installed(self, package_name):
 
 
 def _run(runner, command):
-    print("Running: %s" % command)
+    global _global_output
```

Comment:
I think you might be safe without the "global", as you are not assigning, but just using